org.getmonero.monero.desktop.xml

```diff
@@ -0,0 +1,64 @@
+
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- Copyright 2018-2021 Flathub maintainers -->
+<component type="desktop">
+  <id>org.getmonero.monero.desktop</id>
```

Comment:
id should be `org.getmonero.Monero` (no `.destkop` and case sensitive)

Reply:
I was under the impression that if an application has more than 1 variant, that using a keyword like Desktop or GUI is preferred?
Monero also has a CLI app. And a daemon.

e.g. https://monerodocs.org/interacting/monero-wallet-cli-reference/

I thought this is why apps on Flathub like Bitwarden use the name desktop in the identifier?

e.g. com.bitwarden.desktop
Bitwarden has a Desktop App for 3 different OS. Also, a CLI app, browser extensions, and mobile OS applications.
https://raw.githubusercontent.com/flathub/com.bitwarden.desktop/master/com.bitwarden.desktop.yaml

Reply:
It's not according to the name of the manifest (which has the wrong id anyway). Also it's confusing, but that's just my opinion.